metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/AthenaAggregator.java

```diff
@@ -1,182 +1,195 @@
+                    taskexecutions array<struct<
+                        executionid:string,
+                        taskexecutions:array<struct<
+                            executionid:string,
+                            dateexecuted:string,
+                            executionstatus:string,
+                            executiontime:string,
+                            executiontimeseconds:integer,
+                            memoryrequirementsgb:double,
+                            cpurequirements:int,
+                            cost:struct<value:double,currency:string>,
+                            region:string,
+                            additionalproperties:string
+                            >
+                        >
+                    >>,
+                    validationexecutions array<struct<
+                        executionid:string,
+                        dateexecuted:string,
+                        validatortool:string,
+                        validatortoolversion:string,
+                        isvalid:boolean,
+                        errormessage:string,
+                        additionalproperties:string
+                        >
+                    >
+                )
+                PARTITIONED BY (
+                    `entity` string,
+                    `registry` string,
+                    `org` string,
+                    `name` string,
+                    `version` string,
+                    `platform` string
+                )
+                ROW FORMAT SERDE "org.openx.data.jsonserde.JsonSerDe"
+                LOCATION "s3://{1}/"
+                TBLPROPERTIES (
+                    "projection.enabled" = "true",
+                    "projection.entity.type" = "enum",
+                    "projection.entity.values" = "{2}",
+                    "projection.registry.type" = "enum",
+                    "projection.registry.values" = "{3}",
+                    "projection.org.type" = "injected",
+                    "projection.name.type" = "injected",
+                    "projection.version.type" = "injected",
+                    "projection.platform.type" = "enum",
+                    "projection.platform.values" = "{4}",
+                    "storage.location.template" = "s3://{1}/$'{entity}'/$'{registry}'/$'{org}'/$'{name}'/$'{version}'/$'{platform}'/"
+                )
+                """, tableName, metricsBucketName, entityProjectionValues, registryProjectionValues, platformProjectionValues);
 
-    public Optional<Integer> getCountColumnValue(QueryResultRow queryResultRow) {
-        Optional<Integer> countColumnValue = queryResultRow.getColumnValue(getCountColumnName()).map(Integer::valueOf);
-        if (countColumnValue.isPresent() && countColumnValue.get() == 0) { // There were 0 non-null column values
-            return Optional.empty();
+        try {
+            metricsAggregatorAthenaClient.executeQuery(query);
+        } catch (Exception e) {
```

Comment:
Really general, without looking at the code are there different kinds of exceptions that could be addressed differently?

Reply:
I wanted to catch Athena related errors that can occur during query execution. Looking at the javadoc for the athena client methods called, catching AwsServiceException and SdkClientException should cover it, so I'll change it to that so unrelated errors can be propogated